docs/platforms/flutter/upload-debug.mdx

```diff
@@ -120,6 +126,18 @@ Release commits integration. This is a `string` type with default value `auto`.
 
 Ignore missing commits previously used in the release. This is a boolean type with default value `false`.
 
+`bin_dir`
+
+The folder where the plugin downloads the sentry-cli binary. This is a `string` type with default value `.dart_tool/pub/bin/sentry_dart_plugin`.
+
+`bin_path`
+
+Path to the sentry-cli binary to use instead of downloading. Make sure to use the correct version. This is a `string` type and is empty per default.
+
+`sentry_cli_cdn_url`
+
+Alternative place to download sentry-cli. This is a `string` type with default value `https://downloads.sentry-cdn.com/sentry-cli`. Alternativeley, this can also be set with `SENTRYCLI_CDNURL` environment variable.
```

Comment:
```suggestion
Alternative place to download sentry-cli. This is a `string` type with default value `https://downloads.sentry-cdn.com/sentry-cli`. Alternatively, this can also be set with the `SENTRYCLI_CDNURL` environment variable.
```